packages/design-system/src/stories/examples/Form/ExampleForm.stories.tsx

```diff
@@ -180,7 +180,7 @@ export const ExampleForm = () => {
               aria-label=""
               supportingText="Choose time"
               id="timePicker"
-              defaultDropdownTime={defaultTimePickerDropdownTime}
+              defaultTime={true}
```

Comment:
You don't have to pass `{true}`
```suggestion
              defaultTime
```